packages/web/docs/src/pages/docs/gateway/persisted-documents.mdx

```diff
@@ -0,0 +1,57 @@
+import { Callout, Cards, Tabs } from '@theguild/components'
+
+# Persisted Documents
```

Comment:
This skips all the options of providing a custom store etc here;
https://the-guild.dev/graphql/mesh/v1/serve/features/performance/persisted-operations

Reply:
2 options here:
1- demonstrate it in a tab or something similar
2- point users to code examples instead of creating a new title for it. it feels too dense and hard to understand when there's a super long page with a lot of titles

Reply:
Since they are using the same plugin, and configuration option. I don't think this is too much;
https://the-guild.dev/graphql/mesh/v1/serve/features/performance/persisted-operations#extracting-client-operations
Other than that, all the other options are just all about one small paragraph as you can see here (similar to allowing arbitrary operations)
the-guild.dev/graphql/mesh/v1/serve/features/performance/persisted-operations